spec/serializers/post_serializer_spec

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+
+describe Export::PostSerializer do
+  let(:post) { create(:status_message_with_photo) }
+  subject(:json_output) { Export::PostSerializer.new(post).to_json }
+  subject(:json_output) { Export::PostSerializer.new(post).to_json }
+
+  it { is_expected.to include("\"guid\":\"#{post.guid}\"") }
+  it { is_expected.to include("\"text\":\"#{post.text}\"") }
+  it { is_expected.to include("\"public\":#{post.public}") }
+  it { is_expected.to include("\"diaspora_handle\":\"#{post.diaspora_handle}\"") }
+  it { is_expected.to include("\"type\":\"#{post.type}\"") }
+  it { is_expected.to include("\"image_url\":#{post.image_url}") }
+  it { is_expected.to include("\"image_height\":#{post.image_height}") }
+  it { is_expected.to include("\"image_width\":#{post.image_width}") }
+  it { is_expected.to include("\"likes_count\":#{post.likes_count}") }
+  it { is_expected.to include("\"comments_count\":#{post.comments_count}") }
+  it { is_expected.to include("\"reshares_count\":#{post.reshares_count}") }
+  it { is_expected.to include("\"created_at\":\"#{post.created_at.to_s[0, 4]}") }
```

Comment:
``post.created_at.to_s`` is perfect but ``Export::PostSerializer.new(post).to_json`` results in:

```
expected "{\"post\":{\"guid\":\"15b5e700096d0133dc4548d705df9e7b\",\"text\":\"There are 10 ninjas in this photo.\",\"public\":false,\"diaspora_handle\":\"bob-person-1044ae13@example.net\",\"type\":\"StatusMessage\",\"image_url\":null,\"image_height\":null,\"image_width\":null,\"likes_count\":0,\"comments_count\":0,\"reshares_count\":0,\"created_at\":\"2015-07-10T20:06:33.300Z\"}}" to include "\"created_at\":\"2015-07-10 20:06:33 UTC\""
```